prep-javascript-arrays/main.js

```diff
@@ -0,0 +1,14 @@
+const languages = ['html', 'css', 'javascript', 'react', 'node', 'postgres'];
+console.log(languages);
+var declareFirstElement = 'The first element of the array is';
+var firstElement = languages[0];
+console.log(declareFirstElement, firstElement);
+var declareThirdElement = 'The 3rd element of the array is';
+var thirdElement = languages[2];
+console.log(declareThirdElement, thirdElement);
+var declareLength = 'The length of the array is';
+var length = 6;
+console.log(declareLength, length);
+var declareLastElement = 'The last element of the array is';
+var lastElement = languages[5];
```

Comment:
This will work in this exact situation but we usually don't know the exact last index in our code so we have to calculate the last index using the `length` property from the array. Here is an example:

```JAVASCRIPT
var lastElement = languages[languages.length - 1];
```

Reply:
updated!

<img width="1440" alt="Screen Shot 2021-09-23 at 1 59 01 PM" src="https://user-images.githubusercontent.com/88061673/134559751-25eb5035-5c07-4c55-94f3-2d2d2b0e7eea.png">